modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainer.java

```diff
@@ -37,7 +37,7 @@ public class KafkaContainer extends GenericContainer<KafkaContainer> {
 
     private static final String DEFAULT_INTERNAL_TOPIC_RF = "1";
 
-    private static final String STARTER_SCRIPT = "/testcontainers_start.sh";
+    private static final String STARTER_SCRIPT = "/tmp/testcontainers_start.sh";
```

Comment:
Maybe it would be possible to allow for the path to be fully configurable from the outside instead of hard coding it to `/tmp` ?